draft-ietf-quic-transport.md

```diff
@@ -606,13 +614,41 @@ packet containing the initial cryptographic message needs to be created, this
 includes the packets sent after receiving a Version Negotiation
 ({{packet-version}}) or Retry packet ({{packet-retry}}).
 
+When a server receives an Initial packet with an address validation token, it
+should attempt to validate it.  If the token is invalid then it should just be
+ignored.  If the validation succeeds, the server should then allow future
+connection handshakes to proceed (see {{stateless-retry}}).
+
 
 ### Retry Packet {#packet-retry}
 
-A Retry packet uses long headers with a type value of 0x7E.  It carries
-an opaque token from the server to be used for address validation of the client.
-It is used by a server that wishes to perform a stateless retry
-(see {{stateless-retry}}).
+A Retry packet uses long headers with a type value of 0x7E.  It carries address
+validation token created by the server.  It is used by a server that wishes to
+perform a stateless retry (see {{stateless-retry}}).
+
+~~~
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|     DCIL(8)   |      Original Destination Connection ID (*)   |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                        Retry Token (*)
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+~~~
+
+A Retry packet does not follow the same payload encryption scheme as the Initial
```

Comment:
This is confusing to me. Maybe let's just say that it is not protected under handshake encryption?

Reply:
Well, it's not encrypted at all.